src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -16,16 +32,180 @@ interface ComponentProps {
     title: string;
     startDate?: string;
     endDate?: string;
+    activeProject: number;
+    analysisId: number;
+    setAnalysisDeleteId: () => number;
+}
+
+interface AnalysisPillars {
+    id: number;
+    assigneeName: string;
+    title: string;
+    mainStatement: string;
+    informationGap: string;
+    filters?: string;
+    analysis: number;
+}
+
+interface AnalysisPillarRendererProps extends Omit<AnalysisPillars, 'id'> {
+    pillarId: AnalysisPillars['id'];
+    setDeleteId: () => number;
 }
 
+const mapStateToProps = (state: AppState) => ({
+    activeProject: activeProjectIdFromStateSelector(state),
+});
+
+const MAX_ITEMS_PER_PAGE = 50;
+const AnalysisPillarRenderer = (props: AnalysisPillarRendererProps) => {
+    const {
+        pillarId,
+        title,
+        assigneeName,
+        setDeleteId,
+    } = props;
+
+    const handleDeletePillar = useCallback(() => {
+        setDeleteId(pillarId);
+    }, [pillarId]);
+
+    return (
+        <Container
+            className={styles.pillar}
+            heading={(
+                <div className={styles.left}>
+                    {title}
+                    <Tag
+                        className={styles.tag}
+                        variant="accent"
+                    >
+                        Analysis Completed
+
```

Comment:
usage of translations is recommended

Reply:
This is a dummy text for now which is why no translations were used. After the work for analytical statements is complete, these texts will be received through API. Do we still need to use translations?

Reply:
@subinasr , this should come from translation, as it will be used later on.
```suggestion
                        {complete ? _ts('analysis', 'completeLabel') : _ts('analyss', 'inProgressLabel'))}

```

Reply:
Let me know if there are any confusions.